flexget/plugins/cli/series.py

```diff
@@ -109,7 +109,10 @@ def display_summary(options):
         console('ERROR: %s' % str(e))
         return
     if not porcelain:
-        console(footer)
+        if query.count() == 0:
```

Comment:
The python way is to do:
```python
if not query.count():
```
Also, I think this should be before the table construction, since displaying an empty table is weird IMO.
And maybe add some thing like: "No results found. Try using flexget series all to view all known series" to make it more clear.